packages/core-web/src/index.js

```diff
@@ -21,21 +21,25 @@ function scrollToUrlAnchorHeading() {
   }
 }
 
-function detectAndApplyFixedHeaderStyles() {
+function detectAndApplyHeaderStyles() {
   jQuery(':header').each((index, heading) => {
     if (heading.id) {
       jQuery(heading).removeAttr('id'); // to avoid duplicated id problem
     }
   });
 
-  const headerSelector = jQuery('header[fixed]');
-  const isFixed = headerSelector.length !== 0;
-  if (!isFixed) {
+  const fixedSelector = jQuery('header[fixed]');
+  const isFixed = fixedSelector.length !== 0;
+  const stickySelector = jQuery('header[sticky]');
+  const isSticky = stickySelector.length !== 0;
+  const headerSelector = isSticky ? stickySelector : fixedSelector;
+  if (!(isSticky || isFixed)) {
     return;
   }
 
+  let isHidden = false;
   window.addEventListener('resize', () => {
-    if (window.innerWidth > 767 && headerSelector.hasClass('hide-header')) {
+    if (window.innerWidth > 767 && isHidden) {
```

Comment:
Just a heads up, I think will have to update the `fixed-header-padding` height one more time during resize.

Currently there will be a large padding if the header reappears during resize:

https://user-images.githubusercontent.com/47470981/170835030-f8f08933-7558-476a-9d4d-84344f77504e.mp4